content/news/events-april-2025.md

```diff
@@ -0,0 +1,67 @@
+
+<center><i>Carnage of the Roombening from last year's events. Picture credit to <b>kryma</b> edited by <b>mayam</b></i></center>
+
+ <br/>
+
+ The ice is melting and with the reappearance of the long-forgotten grass comes a whole new month of creation and collaboration to take part in. 
+
+ <br/>
+
+### Wednesday Mesh Evening - Wednesday April 2nd @ 19:00 - 22:00
+
+Explore and help build parts of this decentralized digital community filled with games, interesting people, and information. All hosted on physical nodes throughout Montreal. 
+
+The Wednesday Mesh Evening is a Wednesday evening event dedicated to mesh networking in all its forms. From discussing it, researching and testing alternative network topologies, to practical exercises in deploying peer-to-peer infrastructure in FOULAB and around it.
+
+Come discover the wonders of the Mesh Network with **kryma** and friends!
+
+<br/>
+
+### Killed by Dice Presents: Classic Traveller - Sunday April 13th @ 16:00 - 19:00
+
+Diving into the world of sci-fi roleplaying games join the group for a new adventure.
+
+We will have our first session of the monthly Classic Traveller game. There are still some open spots, please contact david AT killedbydice DOT com for more information, AKA **der_moderne_man** on Mattermost.
+
+<br/>
+
+### Sewing Orientation Workshop Night - Thursday April 17th @ 20:00 - 22:00
+
+Want to learn the secrets of the elusive Foulab sewing machine? Fix your jeans, make a pouch, or even learn how to hem this Thursday.
+
+Walking you through basic sewing machine etiquette, watch **kida** make and commentate his creation workflow. Hope to see you there!
+
+<br/>
+
+### Spring Cleaning - Saturday April 19th @ 09:00 - 18:00
+
+Time to reckon with all the dust, debris, and desolate desktops residing in our beautiful space. 
+
+Join Foulab for the wonderful team-building exercise of getting your hands dirty and making clean. The event will be going on all day so bring your game face and come help out. Maybe enjoy some pizza with everyone after.
+
+<br/>
+
+### FOULAB Monthly Meeting - Tuesday April 22nd @ 20:00 - 21:30
+
+This month we will be discussing the workshop, our incoming lock box, and other general infrastructure information. Come to get a sneak peek at potential workshops and events for next month or to have a say in the group decisions of the lab. 
+
+Share your ideas, your projects, and your thoughts on the future of Foulab. Your input is important to our future!
+
+<br/>
+
+### Project Night - Tuesday April 29th @ 19:00 - 22:00
+
+Calling creators as well as spectators, come share and see what your fellow Foulabers have been working on. Ask questions, watch demos, and discuss passions during this quarterly project night where everyone will be enjoying the work of others.
+
+Beginners welcome! If you’re new to the space, come by and see what we’re all about. It’s a great way to meet people and get involved in the community.
```

Comment:
I think this line uses "RIGHT SINGLE QUOTATION MARK" but elsewhere in this page and all other posts on the page use "APOSTROPHE". Could you make it all use ordinary apostrophe characters?

More info: https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html